### ISSUE TITLE
Children of AudioParamMap missing

### ISSUE THREAD

**Issue by teoli2003** [dontcallmedom/mdn-gaps#1]
There may be a good reason, but I think the properties and methods of

https://developer.mozilla.org/en-US/docs/Web/API/AudioParamMap

should be listed as missing. They have a BCD entry and are supported by 3 engines.

Note that they are defined via a `maplike<>` webidl type, but it was difficult to get a correct spec link: https://webaudio.github.io/web-audio-api/#AudioWorkletNode is the closest link I was able to get.

**Comment by dontcallmedom** [comment]
Looking at a few other examples of `maplike` interfaces:
* https://developer.mozilla.org/en-US/docs/Web/API/EventCounts lists the methods and attributes but only to links to the default definition of Map 
* https://developer.mozilla.org/en-US/docs/Web/API/MIDIInputMap https://developer.mozilla.org/en-US/docs/Web/API/MIDIOutputMap https://developer.mozilla.org/en-US/docs/Web/API/MIDIOutputMap don't list the methods and attributes

It feels like there probably should be first and foremost a discussion on how to handle maplike interfaces in MDN pages, before trying to detect gaps. As an aside, I'm not sure there is a lot of value in displaying the BCD-availability of the "inherited" attributes and methods - I think it's safe to say that if a maplike interface is implemented in a browser, they'll get automatically get the relevant Map members.

**Comment by teoli2003** [comment]
Yes, we have a consensus for constructors, serializers and stringifiers; we need to agree how to display maplike and setlike in BCD.
